osgar/tools/lidarview.py

```diff
@@ -614,7 +641,7 @@ def main(args_in=None, startswith=None):
     with Framer(args.logfile, lidar_name=args.lidar, lidar2_name=args.lidar2, pose2d_name=args.pose2d, pose3d_name=args.pose3d,
                 camera_name=args.camera, camera2_name=args.camera2, bbox_name=args.bbox, joint_name=args.joint,
                 keyframes_name=args.keyframes, title_name=args.title) as framer:
-        lidarview(framer, caption_filename=filename, callback=callback, out_video=args.create_video, jump=args.jump)
+        lidarview(framer, caption_filename=filename, callback=callback, callback_img=callback_img(), out_video=args.create_video, jump=args.jump)
```

Comment:
are you sure, it should be `callback_img()`? then the function is called just once??